platform/admin/ui/src/main/webapp/templates/systemUsage.layout.handlebars

```diff
@@ -17,7 +17,8 @@
             <h5 class="modal-title">{{admin.systemUsageTitle}}</h5>
         </div>
         <div class="modal-body">
-            {{{admin.systemUsageMessage}}}
+            <iframe sandbox='allow-same-origin'>
```

Comment:
Do we need to have `allow-same-origin`?

Reply:
Shouldn't need it unless it's making a request for css or something of that nature.

Reply:
We do need it allowed for now because we are using the parent page to inject the html content.

Reply:
Gotcha - It might be worthwhile at some point to consider having an endpoint that returns the html so we can remove this attribute and just set it's src instead of having to inject the html ourselves.